message/validation/const.go

```diff
@@ -11,16 +11,27 @@ const (
 	// clockErrorTolerance is the maximum amount of clock error we expect to see between nodes.
 	clockErrorTolerance = time.Millisecond * 50
 
-	maxConsensusMsgSize                      = 8388608 // TODO: calculate new value
-	maxPartialSignatureMsgSize               = 1952
-	maxPayloadSize                           = maxConsensusMsgSize
-	maxSignedMsgSize                         = 4 + 56 + maxPayloadSize                // Max possible MsgType + MsgID + Data
-	maxEncodedMsgSize                        = maxSignedMsgSize + maxSignedMsgSize/10 // 10% for encoding overhead
+	maxConsensusMsgSize                      = 705240
+	maxPartialSignatureMsgSize               = 8 + 8 + 1000*(96+32+8+8) // 144016
 	allowedRoundsInFuture                    = 1
 	allowedRoundsInPast                      = 2
 	lateSlotAllowance                        = 2
 	rsaSignatureSize                         = 256
-	blsSignatureSize                         = 96
 	syncCommitteeSize                        = 512
 	maxSignaturesInSyncCommitteeContribution = 13
 )
+
+var (
+	maxPayloadSize    = max(maxConsensusMsgSize, maxPartialSignatureMsgSize) // not const because of max TODO: const after Go 1.21
+	maxSignedMsgSize  = 13*256 + 13*8 + 8 + 56 + maxPayloadSize + 5243144
```

Comment:
what is 5243144?

Reply:
add comment at least or const
other numbers here can also be consts

Reply:
it's from spec, done